### DISCUSSION TITLE
Avatars plugin: not render "Name Characters", if no avatar found

### DISCUSSION THREAD

**Discussion by europacafe** [the-djmaze/snappymail#1711] [Q&A]
SnappyMail 2.37.2
Avatars plugin: 1.19

Here is the sample
![image](https://github.com/user-attachments/assets/2467a83c-731f-46bf-b82f-bd518c93f930)

My Avatars plugin setting
![image](https://github.com/user-attachments/assets/3fd3d701-d07c-45cd-b32b-61f0b9721d9f)

If an Avatar is not found, the preferred display should show "Name Characters" rather than a "missing image file" icon.
![image](https://github.com/user-attachments/assets/5cec6fef-e6cb-4c34-9a0d-e1893351f4cf)

**Comment by the-djmaze** [comment]
This should be fixed

**Reply by europacafe** [reply]
Thanks. I saw it has been fixed after upgrading the plugin.
Thanks again for your active support.
